execution_workflows/QAPA/modules/qapa_index.nf

```diff
@@ -0,0 +1,23 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process QAPA_INDEX {
+
+        publishDir "${params.outdir}/qapa", mode: params.publish_dir_mode
+        container  "docker.io/yuukiiwa/qapa:1.3.1"
```

Comment:
Does Nextflow allow building containers from a Dockerfile? Because I think it'd be good to have the Dockerfiles in the directory so as for the code to be self-contained, thus not needing access to external Docker registries/accounts (and potentially run into rate limits)

Reply:
No problem for double commenting!! I replied above!!